plugins/base_extractor.go

```diff
@@ -15,7 +15,9 @@ func NewBaseExtractor(info Info, configRef interface{}) BaseExtractor {
 // Validate checks if the given options is valid for the plugin.
 func (p *BaseExtractor) Validate(config Config) error {
 	if config.URNScope == "" {
-		return ErrEmptyURNScope
+		return InvalidConfigError{Errors: []ConfigError{
```

Comment:
technically `scope` is not part of the config 🤔 .
Should we have different error for this?

Reply:
Without that, you don't get to know which recipe has the issue:

```
recipe error: urn scope is required to generate a unique urn
```

With this change:
```
kafka-to-http: invalid kafka extractor config: urn scope is required to generate unique urn
```

And since the extractor specifically requires the scope, it might be okay.

Reply:
Will that error be misleading? since it has `invalid kafka extractor config` that could point user concern to `config` block.

Reply:
updated as per discussion.